crates/runtime/Cargo.toml

```diff
@@ -17,6 +18,8 @@ ureq.workspace = true
 wasmer.workspace = true
 wasmer-types.workspace = true
 
+calimero-storage = { path = "../storage" }
+
```

Comment:
`calimero-storage` should not be a dependency of the runtime, if you need it for the example, it should be a `dev-dep` instead